website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -84,6 +84,44 @@ Buttons used for a Dropdown (with the chevron icon) can be found in [Dropdown](/
   <Hds::Button @color="secondary" @text="No text" @icon="plus" @isIconOnly={{true}} />
 </Hds::ButtonSet>
 
+### Leading
+
+In most cases, use leading icons. Choose icons that add meaning and clarity to the action described in the link’s text.
+
+<Hds::ButtonSet>
+  <Hds::Button @color="secondary" @text="Unlock" @icon="unlock" @iconPosition="leading" />
+  <Hds::Button @color="secondary" @text="Show more results" @icon="eye" @iconPosition="leading" />
+  <Hds::Button @color="critical" @text="Delete" @icon="trash" @iconPosition="leading" />
+  <Hds::Button @color="tertiary" @text="Previous" @icon="chevron-left" @iconPosition="leading" />
+</Hds::ButtonSet>
+
+!!! Do
+
+Use the leading position when creating or adding a new object.
+
+<Hds::ButtonSet>
+  <Hds::Button @color="primary" @text="Create variable" @icon="plus" @iconPosition="leading" />
+  <Hds::Button @color="secondary" @text="Add repository" @icon="plus" @iconPosition="leading" />
+</Hds::ButtonSet>
+
+!!!
+
+### Trailing
+
+Consider trailing icons when there’s no other meaningful icon or when guiding the user forward through the product.
```

Comment:
This feels a bit misleading. If there's no other meaningful icon, shouldn't they do without an icon?

Reply:
+ 1

Reply:
I changed this paragraph. Some of the content was adapted from the link docs, and I didn't adapt this part well.